<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStatus</class>
 <widget class="QWidget" name="FormStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_pause">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>暂停/继续视频播放</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="src.qrc">
       <normaloff>:/icos/play.ico</normaloff>:/icos/play.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn_stop">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>关闭正在播放的视频</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="src.qrc">
       <normaloff>:/icos/stop.ico</normaloff>:/icos/stop.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="VlcWidgetSeek" name="player_seek" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetSeek</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetSeek.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
